app/src/pages/inside/logsPage/logsPage.jsx

```diff
@@ -2,25 +2,42 @@ import { Component } from 'react';
 import PropTypes from 'prop-types';
 import { connect } from 'react-redux';
 import { PageLayout, PageSection } from 'layouts/pageLayout';
-import { refreshLogPageData } from 'controllers/log';
+import { activeLogIdSelector, historyItemsSelector, refreshLogPageData } from 'controllers/log';
 import { LogToolbar } from './logToolbar';
 import { HistoryLine } from './historyLine';
+import { LogItemInfo } from './logItemInfo';
 
-@connect(null, {
-  refresh: refreshLogPageData,
-})
+@connect(
+  (state) => ({
+    activeItemId: activeLogIdSelector(state),
+    historyItems: historyItemsSelector(state),
+  }),
+  {
+    refresh: refreshLogPageData,
+  },
+)
 export class LogsPage extends Component {
   static propTypes = {
     refresh: PropTypes.func.isRequired,
+    activeItemId: PropTypes.string,
+    historyItems: PropTypes.array,
+  };
+
+  static defaultProps = {
+    activeItemId: '',
+    historyItems: [],
   };
 
   render() {
-    const { refresh } = this.props;
+    const { refresh, activeItemId, historyItems } = this.props;
+    const activeLogItem = historyItems.find((historyItem) => historyItem.id === activeItemId);
```

Comment:
I'm not sure about getting active item. Maybe for it better to use some selector but I didn't find it.

Reply:
I think it makes sense to add a selector for this

Reply:
Maked activeLogSelector.